onnx/backend/test/runner/__init__.py

```diff
@@ -151,7 +152,9 @@ def _prepare_model_data(self, model_test):
         models_dir = os.getenv('ONNX_MODELS',
                                os.path.join(onnx_home, 'models'))
         model_dir = os.path.join(models_dir, model_test.model_name)
-        if not os.path.exists(model_dir):
+        if not os.path.exists(os.path.join(model_dir, 'model.onnx')):
+            if os.path.exists(model_dir):
+                shutil.rmtree(model_dir)
```

Comment:
Better to not delete user's files, move it to say `~/.onnx/models/squeezenet.old.1` (or `.2` if `.1` already exists)

Reply:
Now move it as .old.0, old.1, etc...